routing/fake_edges_container.hpp

```diff
@@ -32,8 +33,10 @@ class FakeEdgesContainer final
   }
 
 private:
-  // finish segment id
+  // Finish segment id.
   uint32_t m_finishId;
+  // Finish segment located in pass-through/non-pass-through area.
```

Comment:
s/located/is located/
s/pass-through/a pass-through/

Reply:
done